### PR TITLE
Updated the README with all the new groups info

### PR DESCRIPTION
**Pull request by torch2424**
Updated the README with all the cool new stuff we are doing: :smile: 

* Changed "meta" to "working group"
* Explained the new groups concept
* Added a getting involved section
* Fixed some of the titles to fold into one another better in the Table of contents
* Removed individual team members from the Table of Contents

### CODE REVIEW

**Comment by torch2424** [comment]
cc @willemneal and @nearmax for the groups section :smile: :+1:

File: README.md [hunk 1]
```diff
@@ -57,7 +50,21 @@ Organizational Descisions can be interpreted as contributions to the general pro
...
+
+## Working Group
+
+The working group is for those who would like to contribute to the project directly. Whether that be through getting advice on technical implementations, showing off a new tool they are building, or just generally talking about the AssemblyScript project and where it is headed.
+
+**Anyone working on anything related to the AssemblyScript project is welcome to participate in the Working Group!**
+
+## Communitty Group
+
+The [community group](https://github.com/AssemblyScript/community-group) is for those who would like to discuss with other communitty members (including members of the working group) using AssemblyScript in their applications. The goal being that they can prioritize their core feature requests and use cases and communicate that to the working group. Which can help reduce duplicating efforts amongst the communitty, and allow the communitty to be collectively more productive!
```
Comment (MaxGraey) [inline README.md:65]: nit: communitty -> community

Comment (torch2424) [reply README.md:65]: Oh yeah I can't spell :joy: I just saw @dcodeIO opened #35 

So we're going to figure out how to merge the two PRs :upside_down_face:

**Comment by torch2424** [comment]
Closing this in favor of #35 . Just going to move some of my points over there and have one PR for this :joy:

File: README.md (hunk 1 above)
Comment (willemneal) [inline README.md:65]:
```suggestion
The [community group](https://github.com/AssemblyScript/community-group) is for those who would like to discuss with other community members (including members of the working group) using AssemblyScript in their applications. Its goal is to communicate core feature requests and use cases to aid in prioritizing the working group's efforts.  Which can help reduce duplicating efforts amongst the community, and allow the community to be collectively more productive!
```

Comment (torch2424) [reply README.md:65]: reflected in #35 :smile: :+1:

**Pull request closed without merging**